### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_MigrationAssessment_DeveloperTool:There is a name declarative table defined for the table.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15770]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG4.1.2;#AINotInScope;#DevTool

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and tab till view/select button and press enter.
7. Tab till database table.
8. Open developer tool using ctrl+shift+I and hover over to the table.

### Actual:
There is a name declarative table defined for the table.
Similar issue is observed with tables throughout the application.

### Expected:
There should be a proper name "database table" defined for it.

### User Impact:
Screen reader users will not know which table is being announced.

**Comment by Yashi1201** [comment]
Closing this bug as the issue is not repro
Version- 1.42.0-insider(user-setup)

![15770_issue fixed](https://user-images.githubusercontent.com/93374043/218379155-46d6ff2f-5b65-433b-a1f7-a0cb61587cd9.png)

**GithubTags**:#Closed;
